modules/javafx.graphics/src/main/native-prism/SwingInterop.c

```diff
@@ -26,12 +26,12 @@
 #include <jni.h>
 
 /*
- * Class com_sun_javafx_embed_swing_newimpl_SwingNodeInteropN
+ * Class com_sun_javafx_application_PlatformImpl
  * Method: overrideNativeWindowHandle
  * Signature (Ljava/lang/Class;JLjava/lang/Runnable)Ljdk.swing.interop.LightweightFrameWrapper;
```

Comment:
The signature should be changed to match the actual method. You can copy / paste from the generated header file.

Also, I recommend including the generated header file in this file (after the include of `jni.h`)

Reply:
The header file wasn't included prior to this change, which was an oversight. It will compile without it, although you will lose the error checking for mismatched signatures. It is a best practice to include it.